src/partials/_calculator.scss

```diff
@@ -43,13 +44,25 @@ h1 {
   padding: 12px;
   top: -900px;
   position: relative;
+
+  transition: all 2s ease-in-out;
+}
+
+.calcActive {
```

Comment:
the naming convention is different from your other class names.
you are using BEM with dash seperated names. Here, you are using camelCase.
Try not to mix conventions.